lib/nimbase.h

```diff
@@ -188,6 +188,8 @@ __AVR__
 
 #  ifdef __cplusplus
 #    define N_LIB_EXPORT  NIM_EXTERNC __declspec(dllexport)
+#  elif __EMSCRIPTEN__
+#    define N_LIB_EXPORT  EMSCRIPTEN_KEEPALIVE __declspec(dllexport)
```

Comment:
```suggestion
```
The defines here are only meant for when targeting native Windows.

Reply:
Sounds good to me